### DISCUSSION TITLE
how to import one more components with loadModule?

### DISCUSSION THREAD

**Discussion by zhangweihust** [FranckFreiburger/vue3-sfc-loader#58] [Q&A] [answered]
it's vue2 and i have 2 components to import like this:

**

                        loadModule('./button-counter/button-counter.vue', options)
				.then(component => new Vue(component).$mount('#app'));

			loadModule('./my-component2/my-component2.vue', options)
			    .then(component => new Vue(component).$mount('#app'));
**

but loadModule dont support array or map to import the both.

and i wish to use button-counter or  my-component2 with html label  in one div like this
**< div id="app" >
   < button-counter ></ button-counter >
  < my-component2 ></ my-component2 >
</ div >**

please help me how to do it, thank you.

**Comment by FranckFreiburger** [comment, marked as answer]
When your main app is loaded through `vue3-sfc-loader`, further components can be loaded using import (or require).
In most of the cases, only one call to `loadModule()` is needed : for your main .vue file.

_main.vue_ :
```

import buttonCounter from './button-counter/button-counter.vue'
import myComponent2 from './button-counter/my-component2.vue'

<template>
  <button-counter/>
  <my-component2/>
</template>

<script>

export default {

  components: {
    'button-counter': buttonCounter,
    'my-component2': myComponent2,
  }
}

</script>

```

see https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/examples.md#nested-components

**Reply by FranckFreiburger** [reply]
... and https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/faq.md#can-i-call-loadmodule-several-times-
